tony-core/src/main/java/com/linkedin/tony/LocalizableResource.java

```diff
@@ -68,8 +84,14 @@ private void parse(FileSystem fs) throws ParseException, IOException {
         throw new ParseException("Failed to parse file: " + path);
     }
 
+    FileSystem localFs = FileSystem.getLocal(new Configuration());
```

Comment:
Please don't create `new Configuration()` here. Pass in a `Configuration` object to this method if needed.

Also, you can move this inside the `if` block on lines 90-92.